conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
@ptgoetz I don't see any interface changes. Its just additional config in absence of it or the default it will continue to work with older implementation of IMetricsConsumer.